mobile/lib/main.dart

```diff
@@ -167,10 +159,9 @@ class ImmichAppState extends ConsumerState<ImmichApp>
       // Android 8 does not support transparent app bars
       final info = await DeviceInfoPlugin().androidInfo;
       if (info.version.sdkInt <= 26) {
-        overlayStyle =
-            MediaQuery.of(context).platformBrightness == Brightness.light
-                ? SystemUiOverlayStyle.light
-                : SystemUiOverlayStyle.dark;
+        overlayStyle = !context.isDarkTheme
```

Comment:
Maybe `context.isDarkTheme ? SystemUiOverlayStyle.dark : SystemUiOverlayStyle.light` is better, to avoid negative logic

Reply:
Yes. Updated 😄